src/EnergyPlus/WindTurbine.cc

```diff
@@ -93,6 +93,18 @@ namespace WindTurbine {
     // Mazharul Islam, David S.K. Ting, and Amir Fartaj. 2008. Aerodynamic Models for Darrieus-type sSraight-bladed
     //     Vertical Axis Wind Turbines. Renewable & Sustainable Energy Reviews, Volume 12, pp.1087-1109
 
+    constexpr std::array<std::string_view, static_cast<int>(ControlType::Num)> ControlNamesUC{
+        "FIXEDSPEEDFIXEDPITCH",
+        "FIXEDSPEEDVARIABLEPITCH",
+        "VARIABLESPEEDFIXEDPITCH",
+        "VARIABLESPEEDVARIABLEPITCH",
+    };
+
+    constexpr std::array<std::string_view, static_cast<int>(RotorType::Num)> RotorNamesUC{
+        "HORIZONTALAXISWINDTURBINE",
+        "VERTICALAXISWINDTURBINE",
+    };
+
```

Comment:
Non-scalar `constexpr` elements on top of `.cc` file. Could even move these to just before when they are used, since they are only used once.

Reply:
I'm running of things to tell you. They grow up so fast.